crates/db_connection_pool/src/odbcpool.rs

```diff
@@ -19,7 +19,7 @@ use crate::dbconnection::odbcconn::{ODBCDbConnection, ODBCParameter};
 use async_trait::async_trait;
 use datafusion_table_providers::sql::db_connection_pool::{DbConnectionPool, JoinPushDown};
 use odbc_api::{Connection, ConnectionOptions, Environment, sys::AttrConnectionPooling};
-use secrecy::{ExposeSecret, SecretBox, SecretString};
```

Comment:
Is this needed?

Reply:
I can only assume it was picked up by clippy?